From 46dbb3bb924a8a61670222dafb117774584c35c8 Mon Sep 17 00:00:00 2001
From: Lee Jones <joneslee@google.com>
Date: Wed, 3 Dec 2025 08:00:17 +0000
Subject: [PATCH 02/15] Revert "UPSTREAM: PCI/pwrctrl: Create pwrctrl devices
 only when CONFIG_PCI_PWRCTRL is enabled"

This reverts commit 399f71322ed7aedea6575c7e4a1875db7150cf28.

Temporary revert to get Pixel's WiFi and MODEM(s) back up and running
whilst we investigate a more permanent solution.

Bug: 464938478
Change-Id: I872b356e17fb1bc317a60b829e24b4a1fe3b0458
Signed-off-by: Lee Jones <joneslee@google.com>
---
 drivers/pci/probe.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/pci/probe.c b/drivers/pci/probe.c
index 61d2f12996d3..1b080e4e423f 100644
--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -2455,7 +2455,6 @@ bool pci_bus_read_dev_vendor_id(struct pci_bus *bus, int devfn, u32 *l,
 }
 EXPORT_SYMBOL(pci_bus_read_dev_vendor_id);
 
-#if IS_ENABLED(CONFIG_PCI_PWRCTRL)
 static struct platform_device *pci_pwrctrl_create_device(struct pci_bus *bus, int devfn)
 {
 	struct pci_host_bridge *host = pci_find_host_bridge(bus);
@@ -2498,12 +2497,6 @@ static struct platform_device *pci_pwrctrl_create_device(struct pci_bus *bus, in
 
 	return NULL;
 }
-#else
-static struct platform_device *pci_pwrctrl_create_device(struct pci_bus *bus, int devfn)
-{
-	return NULL;
-}
-#endif
 
 /*
  * Read the config data for a PCI device, sanity-check it,
-- 
2.50.1

